pytest.ini

```diff
@@ -1,2 +1,4 @@
 [pytest]
-pythonpath = .
\ No newline at end of file
+pythonpath = .
```

Comment:
Is it safe to use "."?

Reply:
I made my additions consistent with what was there before